config.make

```diff
@@ -98,6 +98,10 @@ ifneq ($(findstring gcc, $(CC)),)
     SOEXT ?= so
     SHFLAG ?= -shared
     PREPATH = -Wl,-rpath,
+else ifneq ($(findstring cc, $(CC)),)
+    CFLAGS += -fPIC
+    SOEXT ?= so
+    SHFLAG ?= -shared
```

Comment:
I don't think this will work because cc can be cray, gnu, intel, or Nvidia, depending on the module loaded.

Reply:
Yeah, I agree. Does that previous PR you just reviewed (and I merged) resolve this already anyways?

Reply:
No, I think it is still an issue.